src/Events/ComplaintMessageEvent.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace jdavidbakr\MailTracker\Events;
+
+use jdavidbakr\MailTracker\Model\SentEmail;
+use Illuminate\Queue\SerializesModels;
+
+class ComplaintMessageEvent
+{
+    use SerializesModels;
+
+    public $email_address;
+
+    /**
+     * Create a new event instance.
+     *
+     * @param  email_address  $email_address
+     * @return void
+     */
+    public function __construct($email_address)
```

Comment:
Don't you want to accept the SentEmail $sent_email here, so it's accessible in the constructor if needed? Also curious how this can be tested (I'd love to help, I already have Email sending with SES working but am trying to set up Amazon SNS - any pointers?

Reply:
Yes, I think that would be more helpful in this case.

Reply:
Probably this and the bounce event should contain -both- the email address and the SentEmail.